src/components/sdk/DeviceSelection/DeviceInput.tsx

```diff
@@ -60,7 +51,7 @@ const DeviceInput: React.FC<Props> = ({
       field={Select}
       options={options}
       onChange={selectDevice}
-      value={selectedDevice}
+      value={selectedDeviceId || ''}
```

Comment:
maybe move this default at line 22 above as default prop valued ?

Reply:
yeah, will do